tachyon-master/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu
+
+MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
+
+RUN apt-get update && \
+  apt-get install -y python libnss3 openjdk-7-jdk wget ssh
+
+# download and unpack archive
+RUN wget https://github.com/amplab/tachyon/releases/download/v0.7.1/tachyon-0.7.1-hadoop2.6-bin.tar.gz
+RUN tar xzvf tachyon-0.7.1-hadoop2.6-bin.tar.gz
```

Comment:
is it better or worse to use compound ```RUN``` statements using ```&&``` compared to multiple ```RUN``` statements?  I thought that was true at one point but don't see an explicit recommendation in the current docs.